include/bluetooth/hci_raw.h

```diff
@@ -32,7 +32,88 @@ extern "C" {
  */
 int bt_send(struct net_buf *buf);
 
-/** @brief Enable Bluetooth RAW channel
+enum {
+	/** Passthrough mode
+	 *
+	 *  While in this mode the buffers are passed as is between the stack
+	 *  and the driver.
+	 */
+	BT_RAW_MODE_PASSTHROUGH = 0x00,
+
+	/** H:4 mode
+	 *
+	 *  While in this mode H:4 headers will added into the buffers
+	 *  according to the buffer type when coming from the stack and will be
+	 *  removed and used to set the buffer type.
+	 */
+	BT_RAW_MODE_H4 = 0x01,
```

Comment:
These seem to be missing "HCI" between "BT" and "RAW".

Reply:
Fixed.